stripe/resource.py

```diff
@@ -6,7 +6,7 @@
 from stripe import api_requestor, error, oauth, util, upload_api_base
 
 
-def convert_to_stripe_object(resp, api_key, account):
+def convert_to_stripe_object(resp, api_key, account, api_version=None):
```

Comment:
Hah, the positional parameter versus named parameter mismatch here isn't amazing, but +1 in that we should be moving toward named, so this is okay.